rmw_zenoh_cpp/src/detail/graph_cache.cpp

```diff
@@ -257,15 +288,19 @@ void GraphCache::parse_del(const std::string & keyexpr)
 
   // Helper lambda to update graph_topics_.
   auto update_graph_topics =
-    [&](const liveliness::TopicInfo topic_info, std::size_t pub_count,
+    [&](const liveliness::TopicInfo topic_info, const EntityType entity_type, std::size_t pub_count,
```

Comment:
I know this was here before, but can we be more explicit about what needs to be captured here?

In point of fact, since this is a method we are calling ourselves, we could possibly get away with no captures and just pass everything we need in as parameters.

Reply:
Done https://github.com/ros2/rmw_zenoh/pull/90/commits/aae76cdfd1a9d5501302116846a6f5db0634359b